queue-manager/rango-preset/src/errors.ts

```diff
@@ -0,0 +1,7 @@
+export class RangoPresetError extends Error {
+  name = 'RangoPresetError';
+
+  constructor(msg: string) {
+    super(msg);
+  }
+}
```

Comment:
I didn't understand point of declaring this class? why just not using Error class?

Reply:
There are two reasons behind this:
1. This approach will helps us to easily check the error using `instanceof` in runtime. 
2. More readable error on sentry. I attached `before` and `after` of doing this.

<img width="299" alt="Screenshot" src="https://github.com/rango-exchange/rango-client/assets/120930547/29f475b9-3642-454c-9b66-ae3959d206ad">
<img width="248" alt="Screenshot" src="https://github.com/rango-exchange/rango-client/assets/120930547/6a0e47cd-c0e8-493f-8045-358e83f9cc54">